mtr/basic/r/mcol2044_signed_bigint_delete-drop-max.result

```diff
@@ -0,0 +1,13 @@
+create database if not exists test_ranges;
+use test_ranges;
+drop table if exists t;
+create table t(x bigint) engine=columnstore;
+insert into t(x) values (44),(55),(66);
+select max_value, min_value from information_schema.columnstore_extents;
+max_value	min_value
+66	44
+delete from t where x=44;
+select max_value, min_value from information_schema.columnstore_extents;
```

Comment:
This query expects columnstore_extent to be empty except the tables created. In real life testing MCS might contain a lot other unexpected extents so I suggest adding a filter.

Reply:
We have other extent-related test cases with same problem. ;)

mcs211_idbExtentId_function.test is one such test.

Reply:
Thank you. GTK.